### PR TITLE
Add a resources page.

### PR DESCRIPTION
**Pull request by hblumberg**
This PR adds a new page with resources from @madeleine-b!

Screen Shot 2021-04-26 at 9 08 28 PM

### CODE REVIEW

File: pages/resources.js [hunk 1]
```diff
@@ -0,0 +1,261 @@
+import React from 'react';
+
+import Layout from '../components/Layout';
+import Link from '../components/subcomponents/Link';
+
+const Resources = () => (
```
Comment (hblumberg) [inline pages/resources.js:6]: If it's a pain to read through the HTML, I'll land to develop, and you can take a look on staging.vaccinatema.com instead (:

File: pages/resources.js [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+                    </li>
+                    <li>
+                        Look at another volunteer-run site{' '}
+                        <Link href={'https://macovidvaccines.com'}>
+                            MA COVID Vaccines
+                        </Link>
+                        .
+                    </li>
+                    <li>
+                        You can also contact MA COVID Vax Help, which is a
```
Comment (madeleine-b) [inline pages/resources.js]: Can we link this to https://macovidvaxhelp.com/?locale=en

File: pages/resources.js [hunk 3]
```diff
@@ -0,0 +1,261 @@
...
+            <p>
+                People with MassHealth or Health Safety Net coverage can get{' '}
+                <Link
+                    href={
+                        'https://mailchi.mp/bostoncil/transportation-services-to-covid-19-vaccine-appointments'
+                    }
+                >
+                    free rides
+                </Link>
+                . People getting vaccines in Arlington, Boston, Brookline,
```
Comment (madeleine-b) [inline pages/resources.js]: Can we add a paragraph break between the BlueBikes program and health insurance one?

**Review by madeleine-b — APPROVED** [review]
Awesome, thank you! Added two comments, okay to merge after those.

**Pull request merged**